boards/limifrog-v1/board.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2015 Hamburg University of Applied Sciences
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_limifrog-v1
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the limifrog-v1 board
+ *
+ * @author      Katja Kirstein <katja.kirstein@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "cpu.h"
+
+static void leds_init(void);
+
+void board_init(void)
+{
+    /* initialize the boards LEDs */
+    leds_init();
+    /* initialize the CPU */
+    cpu_init();
+}
+
+/**
+ * @brief Initialize the boards on-board LEDs
+ *
+ * The LED initialization is hard-coded in this function.
+ * As the LED is soldered onto the board it is fixed to
+ * its CPU pins.
+ *
+ * The red LED is connected to pin PC3
+ */
+static void leds_init(void)
+{
+    /* enable clock for port GPIOA */
```

Comment:
I think it must be GPIO**C** if the LED is connected to PC3 (like stated in line 41)